front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -699,21 +706,37 @@ export default function DataSourcesView({
             </Hoverable>
           </ContentMessage>
         )}
-        <Searchbar
-          ref={searchBarRef}
-          name="search"
-          placeholder="Search (Name)"
-          value={dataSourceSearch}
-          onChange={(s) => {
-            setDataSourceSearch(s);
-          }}
-        />
+        <div className="flex gap-2">
+          {!isAdmin && (
+            <Button
+              label="Request"
+              icon={PlusIcon}
+              onClick={() => setIsRequestDataSourceModalOpen(true)}
+            />
+          )}
```

Comment:
Even admin should see this modal, IMO only the one who originally setup the data source should not see this request modal.

Reply:
Looking at the Figma only users and builders are seeing this button 🤷🏼‍♂️

Reply:
Ok it's actually exposed in the "manage connection" tab, see [here](https://www.figma.com/design/QOxHwppG64GtPocpDhS6Y7/Product?node-id=8487-51611&t=kwmXKVN4foukJtVd-4).